src/Raven.Server/Files/FileSystemFile.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using Sparrow.Json;
+using Voron.Data;
+
+namespace Raven.Server.Files
+{
+    public class FileSystemFile
+    {
+        public long Etag;
+        public LazyStringValue Name;
+        public LazyStringValue LoweredKey;
+        public long StorageId;
+        public BlittableJsonReaderObject Metadata;
+
+        public DateTime LastModifed;
+
+        public VoronStream Stream;
```

Comment:
In order to have also the length. This is not exposed to the client, it is part of the server.